dockerfiles/Dockerfile.cpu

```diff
@@ -2,7 +2,8 @@
 
 FROM ubuntu:16.04
 
-ARG TENSORBOARD_VERSION=0.1.8
+ARG TENSORBOARD_VERSION=1.5.0
+ARG TENSORBOARD_TENSORFLOW_VERSION=1.5.0
```

Comment:
TENSORBOARD_TENSORFLOW_VERSION -> TENSORFLOW_VERSION ? here and in Dockerfile.gpu

Reply:
done